images/base/files/usr/local/bin/entrypoint

```diff
@@ -294,10 +294,12 @@ fix_cgroup() {
   # "nesting" clusters, unless we instruct it to use a different cgroup root.
   # We do this, and when doing so we must fixup this alternative root
   # currently this is hardcoded to be /kubelet
+  # under systemd cgroup driver, kubelet appends .slice
```

Comment:
is there any problem for mounting it twice?
is not possible to detect which one we want?

Reply:
No there's no problem. The cgroup controllers are enabled on many trees. Only one of these two will be used at runtime.

Doing it this way is robust in the face of future changes and user config patches.
We don't know what the node config will be until after we spawn the node and discover details like Kubernetes version.